micromasters/utils.py

```diff
@@ -157,3 +157,19 @@ def is_near_now(time):
     now = datetime.datetime.now(tz=pytz.UTC)
     five_seconds = datetime.timedelta(0, 5)
     return now - five_seconds < time < now + five_seconds
+
+
+def chunks(list_to_chunk, chunk_size=20):
```

Comment:
I know you just copied this code but are we testing it anywhere? If not we should be, maybe in a separate PR